admin/admin.py

```diff
@@ -167,10 +168,17 @@ def index(self):
 
 
 class DocumentView(AuthModelView):
+    def _delete_document(self, model):
+        source = document_path(model.id)
+        if model.has_file and os.path.exists(source):
+            dest = os.path.join(config.DOCUMENT_DIRECTORY, 'trash', str(model.id))
+            while os.path.exists(dest + '.pdf'):
+                dest += 'lol'
```

Comment:
Wait, why would this ever happen? I thought our ids weren't being reused?

Also, may I instead suggest the suffix "lo"?
It looks nicer when repeated